COLLABORATOR_GUIDE.md

```diff
@@ -362,21 +356,14 @@ relies on the commit being available on GitHub.
 git push --tags origin main
 ```
 
-Finally, run `npm publish` with an appropriate tag. Don't forget to supply your token.
-
-```sh
-VJS_GITHUB_USER=gkatsev VJS_GITHUB_TOKEN=my-personal-access-token npm publish --tag next
-```
-
-After it's done, verify that the GitHub release has the correct changelog output.
-This is to make sure that the CHANGELOG didn't get garbled and isn't missing pieces.
+After the tag was pushed, GitHub actions will trigger the `release` workflow, which will do the following:
 
-If the GitHub release did not work correctly, such as if the GitHub token was not provided,
-you can run it manually:
+* Publish to npm with `next` or `next-{n}` depending on your current major version.
+* Create GitHub pre-release with changelog and Netlify preview.
+* Create a GitHub `releases` discussion linked to the GitHub release.
+* Copy files to the CDN with the AWS CLI (this step requires approval, make sure to ping collaborators chat!)
 
-```sh
-VJS_GITHUB_USER=gkatsev VJS_GITHUB_TOKEN=123 node build/gh-release.js --prelease
-```
+And that's it. Congratulations - you've just released a new version of video.js.
```

Comment:
nit(non-blocking):
```suggestion
And that's it. Congratulations - you've just released a new version of Video.js.
```

Feel free to update all *text* occurrences to `Video.js`.

Reply:
updated